hols/langchain4j/.gitignore

```diff
@@ -0,0 +1,39 @@
+# Compiled class file
```

Comment:
If you have new entries add to top level .gitignore instead of creating this file.